### ISSUE TITLE
'Potential Charset conflict' when translate url

### ISSUE THREAD

**Issue by imknown** [Ra-Na/GTranslate-strings-xml#12]
Hello
- Input language: Auto-detect
- Output language: Japanese

Input:
``` xml
<resources>
    ...

    <string name="translator_website">https://imknown.net</string>
    <string name="translator_more_info">https://github.com/imknown</string>

    ...
</resources>
```

Output:
```
https://imknown.net-->[1;31;40mWarning: Potential Charset conflict
 Encoding as extracted by SELF    : Shift_JIS"><meta name="google" content="notranslate"><title>Google 翻訳</title><style>#gbar,#guser{font-size:13px;padding-right:1px;padding-top:1px !important;}#gbar{padding-left:1px;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}@media all{.gb1{height:22px;margin-right:.5em;vertical-align:top}#gbar{float:left}}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb4{color:#00c !important}.gbi .gb4{color:#dd8e27 !important}.gbf .gb4{color:#900 !important}
</style><link rel=stylesheet href="/translate/releases/twsfe_w_20191118_RC00/r/css/desktop_ltr.css"><script></script><script>MSG_GOOGLE_TRANSLATE='Google 翻訳';MSG_SHARING_WEB_TEMPLATE='Google 翻訳で%1$sから%2$sに翻訳されました。';MSG_DISMISS='';MSG_BUTTER_BAR_BUTTON = '';MSG_BUTTER_BAR_DESC = '';BUTTER_BAR_URL = '';BUTTER_BAR_LOGTYPE = '';tld='.com';</script><script>var tn_,tn_aa="function"==typeof ...
```

**Comment by Ra-Na** [comment]
Confirmed

**Comment by Ra-Na** [comment]
Fixed in commit 79853c0
